### DISCUSSION TITLE
Automatic folder creation for new users

### DISCUSSION THREAD

**Discussion by mbendix** [drakkan/sftpgo#1073] [Q&A]
Hello,

I would like to create automatically new folders in the user home directory, if a new user is created. I tried this by using the event manager with an event rule that is trigged by an add-user-provider-event and should execute a create-directories-filesystem-event-action. But nothings happens when I add a new user. Maybe a reason for this behavior could be, that the home directory will be created with the first user logon. Is there an option to create to users home directory directly after the user creation?

Thanks and regards,
Mark

**Comment by drakkan** [comment]
The eventmanager can do this. I no longer provide step-by-step support sorry, please check the [support policy](https://github.com/drakkan/sftpgo#support-policy)
